devops/nightly.yml

```diff
@@ -0,0 +1,79 @@
+trigger: none
+pr: none
+
+schedules:
+  - cron: "30 5 * * *" # Time is UTC
+    displayName: Nightly Code Coverage Build
+    branches:
+      include:
+        - master
+        - dev
+    always: true
+
+pool:
+  vmImage: "ubuntu-latest"
+
+variables:
+  artifactFeed: interpret-extensions-feed
```

Comment:
What do the differences between these builds mean?

Reply:
i.e. job vs steps

Reply:
adding a job is first step to having multiple variants in test (Linux and windows and mac, then python 3.5, 3.6, 3.7) . Each of those will be a job and jobs can be named to reflect the variants while steps can not.